lib/generate-id.js

```diff
@@ -22,11 +22,41 @@ const findId = async ( model, length ) => {
 }
 
 /**
- * Generates a line id and a shabad id, using the database.
+   * Logs the generated id to the console
+   * @param {Number} count the count of id to generate
+   * @param {String} type shabad, line
+   */
+const logGeneratedID = async function ( count, type ) {
+  let i = 0
+
+  if ( type === 'shabad' ) {
+    for ( i = 0; i < count; i += 1 ) {
+      console.log( `${i + 1}. New Shabad ID: `.bold.magenta + `${await findId( Shabads, 3 )}`.rainbow )
+    }
+  } else if ( type === 'line' ) {
+    for ( i = 0; i < count; i += 1 ) {
+      console.log( `${i + 1}. New Line ID: `.bold.magenta + `${await findId( Lines, 4 )}`.rainbow )
+    }
+  } else { // Default log all types
+    for ( i = 0; i < count; i += 1 ) {
+      console.log( `${i + 1}. New Shabad ID: `.bold.magenta + `${await findId( Shabads, 3 )}`.rainbow )
+      console.log( `${i + 1}. New Line ID: `.bold.cyan + `${await findId( Lines, 4 )}`.rainbow )
+    }
+  }
+}
+
+/**
+ * Interpret user command and print the id, using the database.
  */
 const main = async () => {
-  console.log( 'New Line ID: '.bold.cyan + `${await findId( Lines, 4 )}`.rainbow )
-  console.log( 'New Shabad ID: '.bold.magenta + `${await findId( Shabads, 3 )}`.rainbow )
+  commander
+    .version( '0.0.1' )
+    .description( 'Generate ID\'s for ShabadOS Database \n\nCommand \nshabad \t\tgenerate id\'s for shabads \nline \t\tgenerate id\'s for line' )
+    .option( '-c, --count <integer>', 'number of ID\'s', 1 )
+    .option( '-t, --type [command]', 'generate ID for only [command]' )
+    .parse( process.argv )
+
+  await logGeneratedID( commander.count, commander.type )
```

Comment:
A tiny nitpick, but here it makes to destructure the the commander parameters and pass them in

```
const { count, type } = commander

// It's easier in the future to switch out the source of these parameters because of the destructure
await logGeneratedID( count, type)
```